extensions/interactions/DragAndDropSort/directives/drag_and_drop_sort_response_directive.html

```diff
@@ -0,0 +1,35 @@
+<style>
+  .drag-and-drop-sort-response-content img {
+    max-height: 100px;
+  }
+  .item {
+    background-color: #add8e6;
+    border: 1px solid #000000;
+    height: 70px;
+    margin: 10px 0 0 0;
+    width: 70px;
+  }
+  .subitem {
+    background-color: #add8e6;
+    border: 1px solid #000000;
+    height: 70px;
+    margin:-20px 0 0 0;
+    position: relative;
+    width: 70px;
+  }
+</style>
+
+<ol ng-if="answer.length >= 1" class="drag-and-drop-sort-response-content">
```

Comment:
Since it's better to keep the logic in html as minimal as possible, do make an indicator variable in the js file and reference this to that.

Reply:
Done